src/main/asciidoc/override/eventbus.adoc

```diff
@@ -22,3 +22,20 @@ You unregister a message codec with {@link io.vertx.core.eventbus.EventBus#unreg
 
 Message codecs don't always have to encode and decode as the same type. For example you can write a codec that
 allows a MyPOJO class to be sent, but when that message is sent to a handler it arrives as a MyOtherPOJO class.
+
+Vert.x has built-in codecs for certain data types:
+
+- basic types (e.g. strings, booleans, ... etc), or
+- some Vert.x data types (buffers, JSON array, JSON objects), or
+- types implementing the {@link io.vertx.core.shareddata.ClusterSerializable} interface, or
+- types implementing the `java.io.Serializable` interface.
```

Comment:
instead of "types implementing" which implies everything at first read, we should phrase with something like "some of the types" or "a configurable subset of the types implementing serializable"

we should also mention which exact types are allowed by default( byte[] and Number), etc...

Reply:
>instead of "types implementing" which implies everything at first read, we should phrase with something like "some of the types" or "a configurable subset of the types implementing serializable"

All types implementing these interfaces are supported by default, and then there's a note below indicating the restriction in clustered mode.

Reply:
>we should also mention which exact types are allowed by default( byte[] and Number), etc...

done

Reply:
ok